tests/service/Common.fs

```diff
@@ -253,6 +253,19 @@ let getSingleModuleLikeDecl (input: ParsedInput) =
     | ParsedInput.ImplFile (ParsedImplFileInput (modules = [ decl ])) -> decl
     | _ -> failwith "Could not get module decls"
 
+let getSingleModuleMemberDecls (input: ParsedInput) =
+    match getSingleModuleLikeDecl input with
+    | SynModuleOrNamespace (decls = decls) -> decls
+
+let getSingleExprInModule (input: ParsedInput) =
+    match getSingleModuleLikeDecl input with
+    | SynModuleOrNamespace (decls = decls) ->
```

Comment:
Would normally use
```fsharp
let (SynModuleOrNamespace (decls = decls)) = getSingleModuleLikeDecl input
```

Reply:
Why not call `getSingleModuleMemberDecls` here?

Reply:
> Why not call getSingleModuleMemberDecls here?

I think that's what I originally had wanted and didn't finish while were debugging the tests. Thanks, I'll update it.